pymodbus/server/async_io.py

```diff
@@ -556,19 +551,14 @@ class _serverList:
     :meta private:
     """
 
-    active_server: ModbusTcpServer | ModbusUdpServer | ModbusSerialServer
-
-    def __init__(self, server):
-        """Register new server."""
-        self.server = server
-        self.loop = asyncio.get_event_loop()
+    active_server: ModbusTcpServer | ModbusUdpServer | ModbusSerialServer | None
 
     @classmethod
     async def run(cls, server, custom_functions):
         """Help starting/stopping server."""
         for func in custom_functions:
             server.decoder.register(func)
-        cls.active_server = _serverList(server)
+        cls.active_server = server
```

Comment:
This is how we build a list of servers, shortcutting that means no list of servers.